envoy-control-runner/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/infrastructure/ControlPlaneConfig.kt

```diff
@@ -121,6 +124,14 @@ class ControlPlaneConfig {
     ): GlobalServiceChanges =
         GlobalServiceChanges(serviceChanges)
 
+    @Bean
+    fun envoyHttpFilters(
+        properties: EnvoyControlProperties
+    ): EnvoyHttpFilters {
+        val envoyDefaultFilters = EnvoyDefaultFilters(properties.envoy.snapshot)
+        return EnvoyHttpFilters(envoyDefaultFilters.defaultIngressFilters, envoyDefaultFilters.defaultEgressFilters)
```

Comment:
Again, LoD. The config should know the least possible about what the default filters hide underneath.